client/js/libs/handlebars/friendlydate.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const moment = require("moment");
+
+module.exports = function(time) {
+	// See http://momentjs.com/docs/#/displaying/calendar-time/
+	return moment(new Date(time)).calendar(null, {
+		sameDay: "[Today]",
+		lastDay: "[Yesterday]",
+		lastWeek: "L", // Locale
```

Comment:
> I don't think a constant is a much better solution than a comment though

I disagree, I think it's better because apart from anything, we are duplicating the string twice currently, and if we change how we are doing it, we'll have to updatei t in 2 places. I know it's not a big thing, but I'd really like you to do this.